augur/market_info.py

```diff
@@ -1,4 +1,9 @@
-from enum import Enum, auto
+from enum import auto, Enum
+
+from datetime import datetime
+from decimal import Decimal
+
```

Comment:
This file should not change. Only the client should be modified.

Reply:
I changed some property names to match the data we get from the node.